src/test/java/Homework16.java

```diff
@@ -0,0 +1,60 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 {
+    static String user = "salma.ayad.87@gmail.com";
+    static String pass = "te$t$tudent";
+    static String url = "https://bbb.testpro.io/";
+    static WebDriver driver = new ChromeDriver();
+
+    @Test
+    public static void createNewPlaylist() throws InterruptedException {
```

Comment:
no need to use static unless you have a specific goal for it.
public  void createNewPlaylist()